File: app/src/main/java/org/butterbrot/heve/ubill/entity/Fellow.java
package org.butterbrot.heve.ubill.entity;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;

@Entity
public class Fellow {

    @Id
    private Long id;
    private String name;

    public Fellow(String name) {
        this.name = name;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/org/butterbrot/heve/ubill/entity/Fellow.kt
package org.butterbrot.heve.ubill.entity;

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id
import io.objectbox.annotation.Index

@Entity
class Fellow(@Index var name: String = "") {

    @Id
    var id: Long = 0

    override fun toString(): String {
        return name
    }
}